docker/Dockerfile.buster

```diff
@@ -0,0 +1,26 @@
+FROM python:3.7-buster AS builder
+
+RUN useradd --user-group --create-home zulip 
+# && apt-get update \
+# && apt-get install -y libffi-dev libxml2-dev libxslt1-dev 
+
+USER zulip
+WORKDIR /home/zulip
+
+RUN set -ex; python3 -m venv zulip-terminal-venv \
+&& . zulip-terminal-venv/bin/activate \
+&& pip3 --disable-pip-version-check install zulip-term \
+&& rm -rf /home/zulip/.cache
+
+FROM python:3.7-slim-buster
+
+RUN useradd --user-group --create-home zulip \
+&& apt-get update \
+&& apt-get install -y libxml2 libxslt1.1 \
```

Comment:
It is maybe tricky to test this, since many systems depend on this libraries and so it seems inadvisable to remove them, but do the binary wheels of our dependencies include these files in any case?

Reply:
To follow-up, I don't believe we need lines 18 & 19?

Reply:
yes, I already removed them with the following commit

Reply:
I was referring to the libraries, not the dev files.